File: acmICPC.cpp
#include<iostream>
#include<algorithm>
using namespace std;

int main (){
	int arr[6];

	for(int i=0;i<6;i++){
		cin >> arr[i];
	}

	sort(arr,arr+6);

	int sum = 0;
	for(int i=0;i<6;i++){
		sum += arr[i];
	}

	int check = sum/2 - arr[0];
	for(int i=1;i<5;i++){

		int temp = arr[i];
		for(int j=i+1;j<6;j++){
			temp += arr[j];
			
			if(check == temp) {
			cout << "YES";
			return 0;
			}
		}
	}

	cout << "NO";
	return 0;
}

File: acmICPC.js
// var acmICPC = (arr)=>{

// 	arr.sort(function(a,b){return a-b});
// 	var sum  = arr.reduce(function(a,b){return a+b});

// 	if(arr[0]+arr[arr.length-1] > sum /2) return 'NO';
// 	var temp = arr[0]+arr[arr.length-1];

// 	for(var i=1;i< arr.length-2;i++){
// 		if(arr[i]==(sum/2)-temp) return 'YES';
// 	}

// 	return 'NO';
// }

// var arr = [1,3,2,1,2,1];

var arr  = [71 ,66, 124, 199, 67, 147];

arr.sort(function(a,b){return a-b});

console.log(arr)